mteb/tasks/Retrieval/multilingual/MLQARetrieval.py

```diff
@@ -132,31 +132,32 @@ def load_data(self, **kwargs):
                 name=hf_subset,
                 **self.metadata_dict["dataset"],
             )
-            _dataset_raw[lang_pair] = _dataset_raw[lang_pair].rename_column(
-                "context", "text"
-            )
 
-            self.queries[lang_pair] = {
-                eval_split: {
-                    str(i): q["question"]
-                    for i, q in enumerate(_dataset_raw[lang_pair][eval_split])
-                }
-                for eval_split in self.metadata_dict["eval_splits"]
-            }
+            self.queries[lang_pair] = {}
+            self.corpus[lang_pair] = {}
+            self.relevant_docs[lang_pair] = {}
 
-            self.corpus[lang_pair] = {
-                eval_split: {
-                    str(row["id"]): row for row in _dataset_raw[lang_pair][eval_split]
-                }
-                for eval_split in self.metadata_dict["eval_splits"]
-            }
+            for eval_split in self.metadata_dict["eval_splits"]:
```

Comment:
```suggestion
            for eval_split in self.metadata.eval_splits:
```

we are trying to move away from using the dict.